doc/neps/nep-0048-spending-project-funds.rst

```diff
@@ -0,0 +1,421 @@
+.. _NEP48:
+
+=====================================
+NEP 48 — Spending NumPy Project Funds
+=====================================
+
+:Author: Ralf Gommers <ralf.gommers@gmail.com>
+:Author: Inessa Pawson <inessa@albuscode.org>
+:Author: Stefan van der Walt <stefanv@berkeley.edu>
+:Status: Draft
+:Type: Informational
+:Created: 2021-02-07
+:Resolution:
+
+
+Abstract
+--------
+
+The NumPy project has historically never received significant **unrestricted**
+funding. However, that is starting to change.  This NEP aims to provide
+guidance about spending NumPy project unrestricted funds, by formulating a set
+of principles about *what* to pay for and *who* to pay. It will also touch on
+how decisions regarding spending funds get made, how funds get administered,
+and transparency around these topics.
+
+
+Motivation and Scope
+--------------------
+
+In its 16+ year history, the NumPy project has only spent on the order of
+$10,000 USD of funds that were not restricted to a particular program.  Project
+income of this type has been relying on donations from individuals and, from
+May 2019, recurring monthly contributions from Tidelift. By the end of 2020,
+the Tidelift contributions increased to $3,000/month, and there's also a
+potential for an increase of donations and grants going directly to the
+project. Having a clear set of principles around how to use these funds will
+facilitate spending them fairly and effectively. Additionally, it will make it
+easier to solicit donations and other contributions.
+
+A key assumption this NEP makes is that NumPy remains a largely
+volunteer-driven project, and that the project funds are not enough to employ
+maintainers full-time. If funding increases to the point where that assumption
+is no longer true, this NEP should be updated.
+
+In scope for this NEP are:
+
+- Principles of spending project funds: what to pay for, and who to pay.
+- Describing how NumPy's funds get administered.
+- Describing how decisions to spend funds get proposed and made.
+
+Out of scope for this NEP are:
+
+- Making any decisions about spending project funds on a specific project or
+  activity.
+- Principles for spending funds that are intended for NumPy development, but
+  don't fall in the category of NumPy unrestricted funds. This includes most of
+  the grant funding, which is usually earmarked for certain
+  activities/deliverables and goes to an Institutional Partner rather than
+  directly to the NumPy project, and companies or institutions funding specific
+  features.
+  *Rationale: As a project, we have no direct control over how this work gets
+  executed. In some cases, we may not even know the contributions were funded
+  or done by an employee on work time. (Whether that's the case or not should
+  not change how we approach a contribution).  For grants though, we do expect
+  the PI and funded team to align their work with the project's needs and be
```

Comment:
PI is a niche academic (and mostly US) term and does not mean anything to external fiscal bodies worldwide should they wish to examine this NEP.

Reply:
@ilayn Fair point. I’ll reword it.

Reply:
@rgommers Please edit as follows:
<strike>...the PI and funded team to align their work with the project's needs... </strike>
...the research/project leader and funded team to align their work with the needs of NumPy...